lib/l10n/intl_en.arb

```
@@ -706,6 +707,12 @@
       "counterparty": { "type": "String" }
     }
   },
+  "disputeWaitingForAdmin": "Waiting for admin assignment",
+  "disputeYouOpened": "You opened this dispute",
+  "disputeOpenedAgainstUser": "A dispute was opened against you",
+  "disputeInProgress": "Dispute is being reviewed by an admin",
```

RuboCop (1.76.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)